src/autocomplete.js

```diff
@@ -0,0 +1,193 @@
+const makePrefixes = regex => {
+  if (regex.length === 0) {
+    throw new Error('empty regex')
+  }
+  else if (regex.length === 1) {
+    return regex
+  }
+  const prefix = regex.charAt(0) === ' ' ? '\\s+' : regex.charAt(0)
+  return `${prefix}(?:${makePrefixes(regex.slice(1))})?`
+}
+
+const regexPrefixes = regex => new RegExp(makePrefixes(regex), 'iu')
+
+const getSuggestions = (exp, suggestions) => {
+  const digitsMatch = exp.match(/\d+/u)
+  const digits = digitsMatch ? digitsMatch[0] : ''
+  const rePrefixes = suggestions.map(regexPrefixes)
+  return suggestions.filter(
+    (_suggestion, index) => exp.match(rePrefixes[index])
+  ).map(
+    (_suggestion, index) => exp.replace('\\d+', digits).replace(rePrefixes[index])
+  )
+}
+
+const freq = 'every\\s+\\d+?'
+const freqs = ['every ']
+
+const days = [
+  'monday',
+  'tuesday',
+  'wednesday',
+  'thursday',
+  'friday',
+  'saturday',
+  'sunday'
+]
+const day = days.join('|')
```

Comment:
Why the need for another variable? Can't you just put the `.join()` at the end of the array on the previous line?

Reply:
No, I need both variables days and day